holidays/countries/indonesia.py

```
@@ -43,11 +43,12 @@ class Indonesia(
 ):
     """
     References:
-    - https://en.wikipedia.org/wiki/Public_holidays_in_Indonesia
-    - https://id.wikipedia.org/wiki/Hari_libur_di_Indonesia
-    - https://www.liburnasional.com/sejarah/
-    - https://id.wikipedia.org/wiki/Indonesia_dalam_tahun_1963 (1963-2025)
-    - https://www.timeanddate.com/holidays/indonesia
+    
```

Ruff (0.8.2)

Blank line contains whitespace

Remove whitespace from blank line

(W293)